ihmc-communication/src/main/java/us/ihmc/communication/AutonomyAPI.java

```diff
@@ -4,13 +4,15 @@
 import behavior_msgs.msg.dds.AI2RStatusMessage;
 import behavior_msgs.msg.dds.BehaviorTreeStateMessage;
 import us.ihmc.communication.ros2.ROS2IOTopicPair;
+import us.ihmc.ros2.ROS2QosProfile;
 import us.ihmc.ros2.ROS2Topic;
 
 public final class AutonomyAPI
 {
    public static final String BEAVIOR_TREE_MODULE_NAME = "behavior_tree";
 
-   public static final ROS2Topic<?> BEAVIOR_TREE_MODULE = ROS2Tools.IHMC_ROOT.withModule(BEAVIOR_TREE_MODULE_NAME);
+   public static final ROS2Topic<?> BEAVIOR_TREE_MODULE = ROS2Tools.IHMC_ROOT.withModule(BEAVIOR_TREE_MODULE_NAME)
+                                                                             .withQoS(ROS2QosProfile.RELIABLE());
```

Comment:
Switched to RELIABLE, which should avoid dropped messages and resulting errors, even though we handle those and it won't break if they are.